src/index.js

```diff
@@ -1,11 +1,11 @@
 import 'better-log/install';
 
-module.exports = function ({ Plugin, types: t }) {
-	return new Plugin('hello-world', {
+module.exports = function ({ types: t }) {
+	return {
 		visitor: {
-			Program(node, parent, scope, file) {
-				this.unshiftContainer('body', t.expressionStatement(t.literal('use helloworld')));
+			Program(path, file) {
+				path.unshiftContainer('body', t.expressionStatement(t.stringLiteral('use helloworld')));
```

Comment:
> no one was interested in interop

That's a weird view on things, but given how many times this was discussed, I don't really want to start another shitstorm here.